### PR TITLE
Fixes #18350: Document Health Check in user doc

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/18350

### CODE REVIEW

File: src/reference/modules/administration/pages/server.adoc [hunk 1]
```diff
@@ -211,6 +211,20 @@ Event Categories::
 * Policy Node
 * Archives
 
+== Health Check
+To ensure the health of RUDDER, some checks are executed every 5 minutes by default. To change the period
```
Comment (amousset) [inline src/reference/modules/administration/pages/server.adoc:215]:
```suggestion
To ensure the health of Rudder, some checks are executed every 5 minutes by default. To change the period
```

Comment (amousset) [reply src/reference/modules/administration/pages/server.adoc:215]: Let's try to be locally consistent for now, we'll get back on the global level at some point

File: src/reference/modules/administration/pages/server.adoc [hunk 2]
```diff
@@ -211,6 +211,20 @@ Event Categories::
...
+you can add `metrics.healthcheck.scheduler.period` in `/opt/rudder/etc/rudder-web.properties`. For example,
+you can change it to "30 seconds". Every check will return a message with a `Severity Level`
```
Comment (amousset) [inline src/reference/modules/administration/pages/server.adoc:217]: return how?

File: src/reference/modules/administration/pages/server.adoc [hunk 3]
```diff
@@ -211,6 +211,20 @@ Event Categories::
...
+
+Severity Level::
+* Critical: Prevent the proper functioning of RUDDER.
+* Warning: Should be improved, it may cause incidents in the future.
+* Ok: The check was successful.
+
+A notification will be displayed if at least one *Warning* or *Critical* has been returned by a Health Check run.
```
Comment (amousset) [inline src/reference/modules/administration/pages/server.adoc:225]:
```suggestion
A notification will be displayed if at least one *Warning* or *Critical* has been returned by latest Health Check run.
```

Comment (amousset) [reply src/reference/modules/administration/pages/server.adoc:225]: If I have understood correctly

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
Commit modified

**Review by amousset — CHANGES_REQUESTED** [review]

**Comment by ElaadF** [comment]
Commit modified

**Review by amousset — APPROVED** [review]

**Pull request merged**